### DISCUSSION TITLE
Runway Blueprint

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5491] [Q&A] [answered]
When i create blueprint for any resource using runway and try to edit this blueprint and remove the sidebar section after saving it display it again and not removed, how to force it to be removed ?

**Comment by jasonvarga** [comment, marked as answer]
Please open addon issues on their respective repos.

https://github.com/doublethreedigital/runway

**Reply by HazemMohamed252** [reply]
Thanks
